packages/postgres-database/src/simcore_postgres_database/migration/versions/50af59b6c37a_move_projects_workbench.py

```diff
@@ -0,0 +1,103 @@
+"""Move projects workbench
+
+Revision ID: 50af59b6c37a
+Revises: 307017ee1a49
+Create Date: 2025-01-15 12:39:46.219516+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '50af59b6c37a'
+down_revision = '307017ee1a49'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Q: something is not clear to me, how will the tables be kept in sync, when I make a modification to a node, what guarantees it is propagated to their table?

Reply:
In this first version, we agreed to double-write (old + new)